packages/core/src/gqty/index.ts

```diff
@@ -83,10 +84,10 @@ export function createAuthQueryFetcher(
       token = getAccessToken();
     }
 
-    init.headers = Object.assign(
-      { Authorization: `Bearer ${token as string}` },
-      init.headers,
-    );
+    init.headers = {
```

Comment:
We'll want to make sure functionality is still perserved here since this is an actual code change, not just formatting.

Reply:
Lets revert this one back as the differences are not the same:
https://stackoverflow.com/questions/32925460/object-spread-vs-object-assign

Reply:
Ah, this was one of the auto fixed items from the `--fix` command. Thank you so much for catching this guy.